test/utils/image/manifest.go

```diff
@@ -99,6 +99,7 @@ var (
 	BusyBox                  = Config{dockerLibraryRegistry, "busybox", "1.29"}
 	CheckMetadataConcealment = Config{e2eRegistry, "metadata-concealment", "1.2"}
 	CudaVectorAdd            = Config{e2eRegistry, "cuda-vector-add", "1.0"}
+	CudaVectorAdd2           = Config{e2eRegistry, "cuda-vector-add", "2.0"}
```

Comment:
Why this new entry? why not modify version for above entry - CudaVectorAdd?

Reply:
We want to test both Cuda versions. Nvidia drivers should be backward compatible, and we want to make sure user workloads built with older Cuda versions can still run successfully.